creator-node/src/routes/nodeSync.js

```diff
@@ -24,56 +24,92 @@ module.exports = function (app) {
    * }
    */
   app.get('/export', handleResponse(async (req, res) => {
+    // TODO - allow for offsets in the /export
+    const limit = 25000
     const walletPublicKeys = req.query.wallet_public_key // array
     const dbOnlySync = (req.query.db_only_sync === true || req.query.db_only_sync === 'true')
 
     const transaction = await models.sequelize.transaction()
     try {
       // Fetch cnodeUser for each walletPublicKey.
-      const cnodeUsers = await models.CNodeUser.findAll({ where: { walletPublicKey: walletPublicKeys }, transaction })
+      const cnodeUsers = await models.CNodeUser.findAll({ where: { walletPublicKey: walletPublicKeys }, transaction, raw: true })
       const cnodeUserUUIDs = cnodeUsers.map((cnodeUser) => cnodeUser.cnodeUserUUID)
+      const cnodeUserUUID = cnodeUserUUIDs[0] // assume we only have a single wallet being exported for now
 
       // Fetch all data for cnodeUserUUIDs: audiusUsers, tracks, files, clockRecords.
       const [audiusUsers, tracks, files, clockRecords] = await Promise.all([
-        models.AudiusUser.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction }),
-        models.Track.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction }),
-        models.File.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction }),
-        models.ClockRecord.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction })
+        models.AudiusUser.findAll({
+          where: {
+            cnodeUserUUID: cnodeUserUUIDs,
```

Comment:
should these be `cnodeUserUUID` instead of `cnodeUSerUUIDs` (plural) everywhere given line 37 above?

Reply:
yep will make these consistent

Reply:
reverted